### PR TITLE
migration #775 and bug #783: Added postInfo field in TelegramGroup:

### LINKED ISSUE #775: Add postInfo field in TelegramGroup
At @Kulunchick request, we should add _postInfo_ field to TelegramGroup entity. 

image

It should be **boolean** and mean whether to write messages about classes only there. The default value is **false**.
Also add it to TelegramGroup mappers so that all endpoints return it. 

_Don't forget Swagger :)_

### PR DESCRIPTION
**Pull request by JessFreak**
- added postInfo in DbTelegramGroup
- added postInfo in TelegramGroupDTOs
- created checkTelegramGroup in TelegramGroupService
- fixed getGroupsByTelegramId in TelegramGroupMapper
- changed Swagger documentation

closes #775, #783

### CODE REVIEW

File: src/v2/api/controllers/TelegramGroupController.ts [hunk 1]
```diff
@@ -47,6 +47,8 @@ export class TelegramGroupController {
       Source can not be empty
       Source is not an enum
       ThreadId is not a number
+      PostInfo can not be empty
+      PostInfo is not a boolean
       
     ObjectIsRequiredException
```
Comment (ArtemLyah) [inline src/v2/api/controllers/TelegramGroupController.ts:53]: missed ":" write "ObjectIsRequiredException:"

File: src/v2/api/controllers/TelegramGroupController.ts [hunk 2]
```diff
@@ -82,10 +84,14 @@ export class TelegramGroupController {
...
       Source is not an enum
       ThreadId is not a number
+      PostInfo is not a boolean
     
     InvalidEntityIdException:
       TelegramGroup with such id is not found
-      Group with such id is not found`,
+      Group with such id is not found
+      
+      DataNotFoundException
```
Comment (ArtemLyah) [inline src/v2/api/controllers/TelegramGroupController.ts:93]: Remove one tab and add ":"

File: src/v2/api/controllers/TelegramGroupController.ts [hunk 3]
```diff
@@ -113,7 +119,10 @@ export class TelegramGroupController {
     description: `\n
     InvalidEntityIdException:
       TelegramGroup with such id is not found
-      Group with such id is not found`,
+      Group with such id is not found
+      
+    DataNotFoundException
```
Comment (ArtemLyah) [inline src/v2/api/controllers/TelegramGroupController.ts:124]: add ":"

**Review by ArtemLyah — APPROVED** [review]

File: prisma/schema.prisma [hunk 4]
```diff
@@ -62,6 +62,7 @@ model TelegramGroup {
   telegramId BigInt         @map("telegram_id")
   threadId   BigInt?        @map("thread_id")
   source     TelegramSource
+  postInfo   Boolean        @default(false) @map("post_info")
```
Comment (illiakroshka) [inline prisma/schema.prisma:65]: Add postInfo to this endpoint 
image
image

**Review by illiakroshka — APPROVED** [review]

**Review by fokaaas — APPROVED** [review]

**Pull request merged**